cli/flags.rs

```diff
@@ -155,10 +155,6 @@ impl DenoFlags {
       args.push("--allow-hrtime".to_string());
     }
 
-    if self.allow_env {
-      args.push("--allow-env".to_string());
-    }
-
```

Comment:
Add a comment about how allow_env is implied. Or filter it during install.

Reply:
‘allow_env’ already processed L142-L144

Or am i miss some intentions that it is processed twice?

Reply:
oops! no- thank you!